File: api/apisln/src/productidlookup/models/ContactRequestModel.cs
using System;
using System.ComponentModel.DataAnnotations;
public class ContactRequestModel
{
   

    [Required(ErrorMessage = "Email Address is required.")]
    public string EmailAddress;

    [Required(ErrorMessage = "Name is required.")]
    public string Name;

    [Required(ErrorMessage = "Phone Number is required.")]
    public string PhoneNumber;

    [Required(ErrorMessage = "Subject is required.")]
    public string Subject;

    [Required(ErrorMessage = "Message is required.")]
    public string Message;

    [Required(ErrorMessage = "Message is required.")]
    public string RequestSource;

}

File: mymywebapp/src/app/modals/contactrequest.model.ts
export class contactrequest {
    Name?: string;
    EmailAddress?: string;
    PhoneNumber?: string;
    Subject?: string;
    Message?: string;
    RequestSource?:string;


    constructor(
        Name?: string,
    EmailAddress?: string,
    PhoneNumber?: string,
    Subject?: string,
    Message?: string,
    RequestSource?:string
    
    ) {
        this.EmailAddress	      =EmailAddress ;
        this.Message = Message;
        this.Name = Name;
        this.PhoneNumber = PhoneNumber;
        this.RequestSource = RequestSource;
        this.Subject = Subject;
    }

}